packages/react/src/Bento/Bento.tsx

```diff
@@ -0,0 +1,181 @@
+  propName: string,
+  prop?: ResponsiveColumnIndex | ResponsiveRowIndex | ResponsiveFlow | ResponsiveAlign,
+) => {
+  const classesToMerge: string[] = []
+  if (typeof prop === 'number' || typeof prop === 'string') {
+    classesToMerge.push(styles[`Bento__Item--${propName}-${prop}`])
+  } else if (typeof prop === 'object') {
+    for (const [key, value] of Object.entries(prop)) {
+      classesToMerge.push(styles[`Bento__Item--${key}-${propName}-${value}`])
+    }
+  }
+  return classesToMerge
+}
+
+const Item = ({
+  className,
+  columnStart,
+  columnSpan,
+  rowStart,
+  rowSpan,
+  flow,
+  colorMode = 'light',
+  visualAsBackground = false,
+  children,
+  ...rest
+}: BentoItemProps) => {
+  const bentoItemClassArray = [styles.Bento__Item]
+  bentoItemClassArray.push(
+    ...returnClassBasedOnResponsiveMap('column-span', columnSpan),
+    ...returnClassBasedOnResponsiveMap('row-span', rowSpan),
+    ...returnClassBasedOnResponsiveMap('column-start', columnStart),
+    ...returnClassBasedOnResponsiveMap('row-start', rowStart),
+    ...returnClassBasedOnResponsiveMap('flow', flow),
+  )
+
+  return (
+    <div
+      data-color-mode={colorMode}
+      className={clsx(
+        ...bentoItemClassArray,
+        visualAsBackground && styles[`Bento__Item--visual-as-background`],
+        className,
+      )}
+      {...rest}
+    >
+      {children}
+    </div>
+  )
+}
+
+type BentoContentProps = {
+  leadingVisual?: ReactElement | Icon
+  padding?: 'condensed' | 'normal' | 'spacious'
+  verticalAlign?: ResponsiveAlign
+  horizontalAlign?: ResponsiveAlign
+  fixedBottomLink?: boolean
+} & React.HTMLAttributes<HTMLDivElement> &
+  BaseProps<HTMLDivElement>
+
+const Content = ({
+  children,
+  leadingVisual: LeadingVisual,
+  padding = 'normal',
+  verticalAlign = 'start',
+  horizontalAlign = 'start',
+  fixedBottomLink = false,
+  className,
+  ...rest
+}: BentoContentProps) => {
+  const bentoContentClassArray = [styles.Bento__Content]
+  bentoContentClassArray.push(
+    ...returnClassBasedOnResponsiveMap('verticalAlign', verticalAlign),
+    ...returnClassBasedOnResponsiveMap('horizontalAlign', horizontalAlign),
+  )
+  const HeadingChild = React.Children.toArray(children).find(
+    child => React.isValidElement(child) && child.type === Heading,
+  )
+  const TextChild = React.Children.toArray(children).find(child => React.isValidElement(child) && child.type === Text)
+  const LinkChild = React.Children.toArray(children).find(child => React.isValidElement(child) && child.type === Link)
+  return (
+    <div className={clsx(styles[`Bento-padding--${padding}`], ...bentoContentClassArray, className)} {...rest}>
+      {React.isValidElement(LeadingVisual) &&
+        React.cloneElement(LeadingVisual as React.ReactElement<IconProps>, {
+          size: LeadingVisual['size'] || 32,
+        })}
+      {React.isValidElement(HeadingChild) &&
+        React.cloneElement(HeadingChild as React.ReactElement<HeadingProps>, {
```

Comment:
Are we supporting duo tone heading text? Can we add an example in Storybook for how to do this?

Figma: 

![Uploading Screenshot 2023-08-24 at 09.39.23.png…]()

Reply:
This has been added in the following [SHA](https://github.com/primer/brand/pull/390/commits/679e181b3c465e1266854cdea0245a0c2a9378be)